src/overlays/actors/ovl_En_Bh/z_en_bh.c

```diff
@@ -28,16 +29,93 @@ const ActorInit En_Bh_InitVars = {
     (ActorFunc)EnBh_Draw,
 };
 
-#endif
+void EnBh_Init(Actor* thisx, PlayState* play) {
+    EnBh* this = THIS;
+
+    this->actor.flags &= ~ACTOR_FLAG_1;
+    Actor_SetScale(&this->actor, 0.01f);
+    SkelAnime_InitFlex(play, &this->skelanime, &gEnBhNoneSkel, &gEnBhFlyingAnim, this->jointTable, this->morphTable,
+                       OBJECT_BH_LIMB_MAX);
+    Animation_PlayLoop(&this->skelanime, &gEnBhFlyingAnim);
+    this->actionFunc = func_80C22DEC;
+}
+
+void EnBh_Destroy(Actor* thisx, PlayState* play) {
+}
+
+void func_80C22DEC(EnBh* this, PlayState* play) {
+    f32 xDiff;
+    f32 yDiff;
+    f32 zDiff;
+    f32 dist;
```

Comment:
```suggestion
    f32 xzDist;
```
Personally I'd prefer to specify this is the xz distance.